src/com/palmergames/bukkit/util/SupportUtil.java

```diff
@@ -0,0 +1,194 @@
+package com.palmergames.bukkit.util;
+
+import com.palmergames.bukkit.towny.TownyEconomyHandler;
+import org.bukkit.Bukkit;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.HashMap;
+import java.util.WeakHashMap;
+
+/**
+ * Allows testing support for known unsupported classes
+ * @author GNosii
+ */
+@ApiStatus.Internal
+public class SupportUtil {
```

Comment:
I would like the class name to change to PluginSupportUtil in order to be more descriptive of its purpose.